Gem/Connections/Scenes/ConnectionsScene.swift

```diff
@@ -83,7 +83,7 @@ struct ConnectionsScene: View {
             )
         }
         .sheet(isPresented: $isPresentingScanner) {
-            ScanQRCodeNavigationStack(isPresenting: $isPresentingScanner) {
+            ScanQRCodeNavigationStack() {
```

Comment:
can you add consistent names for all method as `ScanQRCodeNavigationStack(onScanFinished(_:))`